### PR TITLE
Borrowings search box get data

### PR DESCRIPTION
**Pull request by at-phutran2**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: #PBI-4
- Task ID: #4.2

## Impacted Areas in Application
List general components of the application that this PR will affect:

- app/Http/Controllers/Admin/BorrowingController.php
- app/Libraries/Traits/SearchTrait.php
- app/Model/Borrowing.php
- resources/lang/en/borrow.php
- resources/views/backend/books/borrow.blade.php
- tests/Browser/Pages/Books/SortBorrowingsTest.php

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- select count(*) as aggregate from `borrowings` left join `users` on `borrowings`.`user_id` = `users`.`id` left join `books` on `borrowings`.`book_id` = `books`.`id` where `borrowings`.`deleted_at` is null

### CODE REVIEW

File: resources/views/backend/books/borrow.blade.php [hunk 1]
```diff
@@ -68,16 +68,16 @@
...
+                                    <td>{{ $borrowing->book_name }}</td>
                                     <td>{{ date(config('define.date_format'), strtotime($borrowing->from_date)) }}</td>
                                     <td>{{ date(config('define.date_format'), strtotime($borrowing->to_date)) }}</td>
                                     <td>{{ $borrowing->date_send_mail }}</td>
                                     <td>
                                         <button type="button" data-action= "{{ route('sendMail', $borrowing->id) }}" 
-                                        class="btn btn-warning btn-check fa-trash-o ion ion-android-drafts" data-name="{{ $borrowing->users->name }}" data-toggle="modal" data-target="#confirmSendMail" id="{{ $borrowing->id }}" {{ canSendMail($borrowing->date_send_email) ? '' : 'disabled' }}>
+                                        class="btn btn-warning btn-check fa-trash-o ion ion-android-drafts" data-name="{{ $borrowing->name }}" data-toggle="modal" data-target="#confirmSendMail" id="{{ $borrowing->id }}" {{ canSendMail($borrowing->date_send_email) ? '' : 'disabled' }}>
```
Comment (at-duyho) [inline resources/views/backend/books/borrow.blade.php]: $borrowing->name? kiểm tra thì không có select column này.

File: resources/views/backend/books/borrow.blade.php [hunk 2]
```diff
@@ -68,16 +68,16 @@
...
-                                    <td>{{ $borrowing->users->email }}</td>
-                                    <td>{{ $borrowing->books->name }}</td>
+                                    <td>{{ $borrowing->employee_code }}</td>
+                                    <td>{{ $borrowing->user_name }}</td>
+                                    <td>{{ $borrowing->email }}</td>
+                                    <td>{{ $borrowing->book_name }}</td>
                                     <td>{{ date(config('define.date_format'), strtotime($borrowing->from_date)) }}</td>
                                     <td>{{ date(config('define.date_format'), strtotime($borrowing->to_date)) }}</td>
                                     <td>{{ $borrowing->date_send_mail }}</td>
                                     <td>
```
Comment (at-duyho) [inline resources/views/backend/books/borrow.blade.php]: $borrowing->date_send_mail? không thấy có select column này.

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment, thank you!

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**